scripts/examples/gn_efr32_example.sh

```diff
@@ -133,15 +133,19 @@ else
     shift
     while [ $# -gt 0 ]; do
         case $1 in
+            --clean)
+                DIR_CLEAN=true
+                shift
+                ;;
             --wifi)
                 if [ -z "$2" ]; then
                     echo "--wifi requires rs911x or wf200"
                     exit 1
                 fi
                 if [ "$2" = "rs911x" ]; then
-                    optArgs+="use_rs911x=true "
+                    optArgs+="use_rs911x=true"
```

Comment:
need the space here. else you need to a step in the arg parsing to add the space if there is a another argument that is following